packages/server/lib/refreshConnections.ts

```diff
@@ -99,3 +106,15 @@ export async function exec(): Promise<void> {
         }
     });
 }
+
+function connectionTestHook(
+    providerName: string,
+    provider: Provider,
+    credentials: ApiKeyCredentials | BasicApiCredentials | TbaCredentials,
+    connectionId: string,
+    providerConfigKey: string,
+    environment_id: number,
+    connection_config: ConnectionConfig
+): Promise<Result<boolean, NangoError>> {
+    return connectionTest(providerName, provider, credentials, connectionId, providerConfigKey, environment_id, connection_config, tracer);
+}
```

Comment:
needed so I can pass the tracer to `shared`

Reply:
I wonder if it's really useful, maybe tracer usage is not required and we can load dd-trace directly here

Reply:
good point. I though dd-trace was not a dependency of shard but it actually is.

Reply:
I'm not used to tracing so not sure if there is a global state or only one attached to the tracer object